docs/src/content/sections/pricing.md

```diff
@@ -3,109 +3,110 @@ title: "Choose a pricing plan best suited <em>for you</em>"
 
 plans:
   - enable: true
-    title: Trial
-    description: Keen to try it out first? Sign up for a free two weeks trial.
-    price_prefix: ""
+    title: Free
+    description: Explore how our 30+ Xcode Simulator features can increase your productivity
+    price_prefix: "€"
     color: "#1d1d1f"
     features:
       - enable: true
-        included: true
-        feature: Pro access to RocketSim.
+        included: false
+        feature: Pro access to RocketSim
       - enable: true
-        included: true
-        feature: Team build insights.
+        included: false
+        feature: Team build insights
       - enable: true
-        included: true
-        feature: User management.
+        included: false
+        feature: User management
       - enable: true
-        included: true
-        feature: License management.
+        included: false
+        feature: License management
     price:
       yearly:
-        amount: FREE
-        period: ""
+        amount: 0
+        period: per month
     cta:
       enable: true
-      label: Sign up for a free trial
-      site: teams
-      link: "/signup/trial"
+      label: Download now
+      site: custom
+      link: "https://apps.apple.com/app/apple-store/id1504940162?pt=117264678&ct=pricing-table-free&mt=8"
+      class: "plausible-event-name=App+Store+Install+Free"
   - enable: true
-    title: Individual
-    description: Perfect for solo developers.
+    title: Personal
+    description: Ideal for solo developers—enjoy every Pro feature with an in-app purchase
     price_prefix: "€"
     color: "#1d1d1f"
     features:
       - enable: true
         included: true
-        feature: Pro access to RocketSim.
+        feature: Pro access to RocketSim
       - enable: true
         included: false
-        feature: Team build insights.
+        feature: Team build insights
       - enable: true
         included: false
-        feature: User management.
+        feature: User management
       - enable: true
         included: false
-        feature: License management.
+        feature: License management
     price:
       yearly:
-        amount: 50
-        period: per year
+        amount: 5
+        period: per month
```

Comment:
[The theme we're using has a month/year switch.](https://telldesk-astro.vercel.app/pricing) I can bring it back if you want? It might be a bit weird though, because we don't give a discount when payed on a yearly basis.

Reply:
Yeah, let's not do that 🙂 We do have a discount in the App Store btw, but let's not communicate that here